From private repo

v-alje opened the pull request.

azurecla commented:
Hi __@ShawnJackson__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (v-shawja). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

nitinme committed:
Merge remote-tracking branch 'refs/remotes/Microsoft/master' into nitinme-hdi-oms

nitinme committed:
Draft 1 - Get started with OMS + HDI

nitinme committed:
Merge remote-tracking branch 'refs/remotes/Microsoft/master' into nitinme-hdi-oms

nitinme committed:
Merge remote-tracking branch 'refs/remotes/Microsoft/master' into nitinme-hdi-oms

nitinme committed:
Merge remote-tracking branch 'refs/remotes/Microsoft/master' into nitinme-hdi-oms

nitinme committed:
Merge remote-tracking branch 'refs/remotes/Microsoft/master' into nitinme-hdi-oms

nitinme committed:
Adding new article for creating OMS custom views

nitinme committed:
Merge remote-tracking branch 'refs/remotes/Microsoft/master' into nitinme-hdi-oms

nitinme committed:
adding another link for see also

nitinme committed:
Merge remote-tracking branch 'refs/remotes/Microsoft/master' into nitinme-hdi-oms

nitinme committed:
adding article on top queries

nitinme committed:
Merge remote-tracking branch 'refs/remotes/Microsoft/master' into nitinme-hdi-oms

nitinme committed:
Merge branch 'nitinme-hdi-oms' of https://github.com/nitinme/azure-docs-pr into nitinme-hdi-oms

nitinme committed:
another update

nitinme committed:
Merge remote-tracking branch 'refs/remotes/MicrosoftDocs/master' into nitinme-hdi-oms

nitinme committed:
Merge remote-tracking branch 'refs/remotes/MicrosoftDocs/master' into nitinme-hdi-oms

nitinme committed:
Adding more queries

nitinme committed:
Merge remote-tracking branch 'refs/remotes/MicrosoftDocs/master' into nitinme-hdi-oms

nitinme committed:
More queries added

nitinme committed:
Merge remote-tracking branch 'refs/remotes/MicrosoftDocs/master' into nitinme-hdi-oms

nitinme committed:
Merge remote-tracking branch 'refs/remotes/MicrosoftDocs/master' into nitinme-hdi-oms

nitinme committed:
Merge remote-tracking branch 'refs/remotes/MicrosoftDocs/master' into nitinme-hdi-oms

nitinme committed:
Merge remote-tracking branch 'refs/remotes/MicrosoftDocs/master' into nitinme-hdi-oms

nitinme committed:
Merge remote-tracking branch 'refs/remotes/MicrosoftDocs/master' into nitinme-hdi-oms

nitinme committed:
fixing merge conflict

# Conflicts:
#	articles/hdinsight/hdinsight-hadoop-oms-log-analytics-top-queries.md

nitinme committed:
removing the top queries article for the initial release of the feature

nitinme committed:
Merge remote-tracking branch 'refs/remotes/MicrosoftDocs/master' into nitinme-hdi-oms

nitinme committed:
more updates

nitinme committed:
more updates

nitinme committed:
Merge remote-tracking branch 'refs/remotes/MicrosoftDocs/master' into nitinme-hdi-oms

nitinme committed:
Merge branch 'nitinme-hdi-oms' of https://github.com/nitinme/azure-docs-pr into nitinme-hdi-oms

nitinme committed:
Merge remote-tracking branch 'refs/remotes/MicrosoftDocs/master' into nitinme-hdi-oms

nitinme committed:
more updates

nitinme committed:
Merge remote-tracking branch 'refs/remotes/MicrosoftDocs/master' into nitinme-hdi-oms

nitinme committed:
minor updates

nitinme committed:
Merge remote-tracking branch 'refs/remotes/MicrosoftDocs/master' into nitinme-hdi-oms

nitinme committed:
updating some links

nitinme committed:
Merge remote-tracking branch 'refs/remotes/MicrosoftDocs/master' into nitinme-hdi-oms

nitinme committed:
Merge remote-tracking branch 'refs/remotes/MicrosoftDocs/master' into nitinme-hdi-oms

nitinme committed:
Merge remote-tracking branch 'refs/remotes/MicrosoftDocs/master' into nitinme-hdi-oms

nitinme committed:
updating the TOC file

nitinme committed:
formatting fix

nitinme committed:
formatting update

nitinme committed:
fixing indentation

nitinme committed:
Merge remote-tracking branch 'refs/remotes/MicrosoftDocs/master' into nitinme-hdi-oms

Juliako committed:
Update media-services-dotnet-how-to-use-azure-functions.md

ggailey777 committed:
Hide Event Grid + Functions tutorial

ggailey777 committed:
Remove TOC target part--doesn't work

nitinme committed:
Merge remote-tracking branch 'refs/remotes/MicrosoftDocs/master' into nitinme-hdi-oms

nitinme committed:
Last few updates

nitinme committed:
Acrolinx fixes

nitinme committed:
Merge remote-tracking branch 'refs/remotes/MicrosoftDocs/master' into nitinme-hdi-oms

nitinme committed:
Minor TOC change

Commit:
Merge pull request #21821 from Juliako/patch-16

Update media-services-dotnet-how-to-use-azure-functions.md

nberdy committed:
Update quick-setup-auto-provision.md

nberdy committed:
Add files via upload

ktoliver committed:
Merge pull request #16132 from nitinme/nitinme-hdi-oms

Content updates for HDI + OMS

czeumault committed:
Merge pull request #21952 from ggailey777/master

Hide Event Grid + Functions tutorial

v-thepet committed:
Merge pull request #21975 from nberdy/patch-6

Update quick-setup-auto-provision.md

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into FromPrivateRepo
